ros/src/computing/planning/motion/packages/waypoint_follower/launch/mpc_follower.launch

```diff
@@ -0,0 +1,139 @@
+<launch>
+
+	<arg name="ns" default="/"/>
+	<arg name="show_debug_info" default="false"/>
+
+	<!-- control period [s] -->
+	<arg name="ctrl_period" default="0.04"/>
```

Comment:
please use argument description using the `doc` [attribute](http://wiki.ros.org/roslaunch/XML/arg#Attributes)